applications/user-generated-content.md

```diff
@@ -0,0 +1,17 @@
+---
+description: Machine translation for user-generated content
+---
+
+# User-Generated Content
+
+Machine translation for **user-generated content (UGC)** is used mainly for **gisting**.
+User-generated content is a good fit for machine translation because it is very high-scale.
+User-generated content is challenging for machine translation because it is open-domain and noisy.
+
+
+## Application
+Readers can choose to machine translate USG on a sentence-basis.
+
+### Goals
+With machine translation, users widen their audience and engage different communities.
```

Comment:
"engage different communities" doesn't mean much

I think it boils down to 2 things:

>  Translation can drive key platform metrics like user engagement and sales.  But user-generated content is too high-scale for human translation.